pkg/storage/registry/spaces/spaces.go

```diff
@@ -76,7 +77,8 @@ func (sc *spaceConfig) SpacePath(currentUser *userpb.User, space *providerpb.Sto
 // Provider holds information on Spaces
 type Provider struct {
 	// Spaces is a map from space type to space config
-	Spaces map[string]*spaceConfig `mapstructure:"spaces"`
+	Spaces     map[string]*spaceConfig `mapstructure:"spaces"`
+	ProviderID string                  `mapstructure:"providerid"`
```

Comment:
We generate provider IDs dynamically. Can you make these work with regexes?

https://github.com/cs3org/reva/blob/8485d4046dbc4f8b4395d82eea78129b928833bb/pkg/storage/registry/static/static_test.go#L57-L65

For example, provider IDs `1284d238-aa92-42ce-bdc4-0b000000[9157-9159]` should go to `localhost:11000`

Reply:
And doing a `ListProviders` without any filters should ideally return all of these:
[
  {address: localhost:11000, provider_id: 1284d238-aa92-42ce-bdc4-0b0000009157},
  {address: localhost:11000, provider_id: 1284d238-aa92-42ce-bdc4-0b0000009158},
  {address: localhost:11000, provider_id: 1284d238-aa92-42ce-bdc4-0b0000009159}
]

Reply:
I can look into this in a separate PR!